compiler/shared_ast/typing.ml

```diff
@@ -354,13 +354,11 @@ let polymorphic_op_return_type
 let resolve_overload_ret_type
     ~flags
     (ctx : A.decl_ctx)
-    e
-    (op : Operator.overloaded A.operator)
+    _e
```

Comment:
Unused argument that can be deleted?

Reply:
Ah indeed, it was used for position but now the operator includes that